src/content/en/android/trusted-web-activity/quick-start/index.md

```diff
@@ -49,32 +50,75 @@ out, this is called **verification**.
 
-The [svgomg-twa](https://github.com/GoogleChromeLabs/svgomg-twa) repo contains an example
-Android application that launches a Trusted Web Activity. You can customize to launch your website:
+[Bubblewrap](https://github.com/GoogleChromeLabs/bubblewrap) is a set of libraries and a command
+line tool (CLI) for Node.js that helps developers generate, build and run Progressive Web Apps
+inside Android applications, using Trusted Web Activity.
 
-1. Clone the project (`git clone https://github.com/GoogleChromeLabs/svgomg-twa.git`).
-2. Import the Project into Android Studio, using **File** > **New** > **Import Project**, and select
-   the folder to which the project was cloned.
-3. Open the app's `build.gradle` and modify the values in `twaManifest`.
-   There are two `build.gradle` files.
-   You want the module one at `app/build.gradle`.
+The CLI can be installed with the following command:
 
-    * Change `hostName` to point to your website.
-      Your website must be available on HTTPS, though you omit that from the `hostName` field.
-    * Change `name` to whatever you want.
-    * Change `applicationId` to something specific to your project.
-      This translates into the app’s package name and is
-      [how the app is identified](https://developer.android.com/studio/build/application-id) on the
-      Play Store - no two apps can share the `applicationId` and if you change it you’ll need to
-      create a new Play Store listing.
+```shell
+npm i -g @bubblewrap/cli
+```
+
+### Setting up the Environment
+
+#### Get the Java Development Kit (JDK) 8.
+The Android Command line tools requires the correct version of the JDK to run. To prevent version
+conflicts with a JDK version that is already installed, Bubblewrap uses a JDK that can unzipped in
+a separate folder.
+
+Download a version of JDK 8 that is compatible with your OS from
+[AdoptOpenJDK](https://adoptopenjdk.net/releases.html?variant=openjdk8&jvmVariant=hotspot)
+and extract it in its own folder.
+
+**Warning:** Using a version lower than 8 will make it impossible to compile the project and higher
+versions are incompatible with the Android command line tools.
+
+#### Get the Android command line tools
+Download a version of Android command line tools that is compatible with your OS from
+[https://developer.android.com/studio#command-tools](https://developer.android.com/studio#command-tools).
+Create a folder and extract the downloaded file into it. The tool will further install its
+dependencies, without the need to install the whole Android IDE.
+
+When running `bubblewrap` for the first time, it will ask where it can find the JDK and Android
+command line tools. So, take note of the location where both were decompressed.
+
+## Initialize and build project {: initialize-and-build}
+
+Initializing an Android project that wraps a PWA is done by running the init command:
+
+```shell
+bubblewrap init --manifest https://my-twa.com/manifest.json
```

Comment:
Does this not need to start with `npx ...`?

Reply:
In this case, we're running `npm i -g @bubblewrap/cli`, so it's installed and can be run with `bubblewrap`. `npx` allows running the package without installing.